jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemParser_1_0.java

```diff
@@ -56,6 +58,6 @@ public void readElement(final XMLExtendedStreamReader reader, final List<ModelNo
        */
       @Override
       public void writeContent(final XMLExtendedStreamWriter streamWriter, final SubsystemMarshallingContext context) throws XMLStreamException {
-          context.startSubsystemElement(JaxrsExtension.NAMESPACE_1_0, true);
+          context.startSubsystemElement(NAMESPACE, true);
```

Comment:
Since you went down the refactoring path I'm going to punish you. ;)

j/k -- cleanup is good.

This class should no longer implement XMLElementWriter as it is never used as such.

Reply:
Ah good call. I'll fix that as well. It's a quick change.